### PR TITLE
Camunda 2023 submission (bpmn.io 11.5.0)

### PR DESCRIPTION
**Pull request by barmac**
Note: https://github.com/bpmn-miwg/bpmn-miwg-test-suite/pull/893#issuecomment-1450496384

### CODE REVIEW

File: tools-tested-by-miwg.json [hunk 1]
```diff
@@ -116,15 +116,14 @@
...
       "website": "https://camunda.com/bpmn/tool/",
       "bpiId": "",
       "supportsBpmn2": true,
       "hasImport": true,
       "hasExport": true,
       "hasRoundtrip": true,
-      "lastResultsSubmitted": "2022",
-      "participatedInDemosByYear": "2022,2020,2019,2018,2017,2016,2015,2014,2013",
+      "lastResultsSubmitted": "2023",
+      "participatedInDemosByYear": "2023,2022,2020,2019,2018,2017,2016,2015,2014,2013",
```
Comment (falko) [inline tools-tested-by-miwg.json:126]: @barmac There was no demo this year. So we should not yet add an entry to `participatedInDemosByYear`

File: tools-tested-by-miwg.json [hunk 2]
```diff
@@ -116,15 +116,14 @@
       "vendor": "Camunda",
       "tool": "bpmn.io (Camunda Modeler)",
       "maintainer": "nele.uhlemann(at)camunda.com",
```
Comment (falko) [inline tools-tested-by-miwg.json:118]: @barmac would you like to add your email address there?

File: tools-tested-by-miwg.json [hunk 3]
```diff
@@ -116,15 +116,14 @@
...
-      "version": "9.4.0",
```
Comment (falko) [inline tools-tested-by-miwg.json:119]: @barmac The version number is mandatory.

**Review by falko — CHANGES_REQUESTED** [review]

File: tools-tested-by-miwg.json (hunk 3 above)
Comment (barmac) [reply tools-tested-by-miwg.json:119]: It should be 11.5.0. This is a bug in the automation script.

**Review by falko — APPROVED** [review]

**Comment by falko** [comment]
@tstephen could you run the automation on this?

**Comment by tstephen** [comment]
@falko Interesting, since you too are an admin when you merged the automation works. Could be we do have the org token on it after all.

**Pull request merged**